eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -99,6 +100,25 @@ stages:
         overrideDistroDisablingSha1: false # 🚫
         runOnline: true                    # ✅
         withPreviousSDK: true              # ✅
+        useMonoRuntime: false              # 🚫
+
+    - template: ../jobs/vmr-build.yml
+      parameters:
+        buildName: CentOSStream8_Mono_Online_PreviousSourceBuiltSdk
```

Comment:
What is the reason to run online vs offline?

Reply:
I just picked one to start from. We can change it based on the scenarios that make most sense to validate.